meta-facebook/gt-cc/src/platform/plat_fru.h

```diff
@@ -8,11 +8,10 @@
 #define FIO_FRU_PORT 0x04
 #define FIO_FRU_ADDR (0xA2 >> 1)
 
-enum {
-	SWB_FRU_ID,
-	FIO_FRU_ID,
-	// OTHER_FRU_ID,
-	MAX_FRU_ID,
+enum { SWB_FRU_ID,
+       FIO_FRU_ID,
+       // OTHER_FRU_ID,
+       MAX_FRU_ID,
```

Comment:
It's my mistake to do clang-format for non-WaimeaCanyon file. This is GrandTeton's file.

Reply:
No I can fix it internally.

I've added a githook that should automatically check formatting.
It can be found in the scripts directory, you just need to add it to your `.git/hooks/` directory locally.